Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -136,7 +136,8 @@ void GlobalObject::initialize_global_object()
     define_native_function(vm.names.isFinite, is_finite, 1, attr);
     define_native_function(vm.names.parseFloat, parse_float, 1, attr);
     define_native_function(vm.names.parseInt, parse_int, 2, attr);
-    define_native_function(vm.names.eval, eval, 1, attr);
+    m_true_eval_function = define_native_function(vm.names.eval, eval, 1, attr);
```

Comment:
Let's not change `define_native_function` to do this, you can instead just `get_without_side_effects()` and store it after defining.

Also, `m_true_eval_function` is a strange name - just `m_eval_function` is fine.

Reply:
fixed